### ISSUE TITLE
[Question] How can i make focusedDataPointLabelComponent above everything? 

### ISSUE THREAD

**Issue by JosephKorel** [Abhinandan-Kushwaha/react-native-gifted-charts#396]
Hi, i have this `focusedDataPointLabelComponent`
```
focusedDataPointLabelComponent: () => (
        <Box
          style={{ zIndex: 999 }}
          px={4}
          py={1}
          bottom={10}
          right={10}
          bg="white"
          borderWidth={2}
          borderColor="primary.500"
          borderRadius="full"
        >
          <CurrencyText
            value={item[viewing] as number}
            prefix
            decimalScale={2}
            fontFamily="montserrat"
            color="primary.500"
            style={{ fontWeight: "800" }}
          />
        </Box>
      ),
```

And this `dataPointLabelComponent`
```
dataPointLabelComponent: () => {
        if (hasIncreased() == null) {
          return <></>;
        }
        return (
          <Box right={2}>
            {hasIncreased() ? (
              <Icon as={AntDesign} name="arrowup" color="green.500" size="lg" />
            ) : (
              <Icon as={AntDesign} name="arrowdown" color="red.400" size="lg" />
            )}
          </Box>
        );
      },
```

The only problem is that the Icon is above the rendered tooltip. I tried passing `zIndex` to it, but it didn't work. Is there a way to make it always above everything? 
This is how it looks like
![image](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/96602536/09f2253b-caf7-4d18-a63a-263e50b5b122)

**Comment by Abhinandan-Kushwaha** [comment]
Hi @JosephKorel 👋
Thanks for bringing this up. I will look into it and give a solution by tonight.

**Comment by Abhinandan-Kushwaha** [comment]
Hi @JosephKorel 👋
This has been fixed in version `1.3.9`🎉
Please update.
